composer.json

```diff
@@ -38,9 +38,10 @@
         "swiftmailer/swiftmailer": "^6.2",
         "symfony/cache": "^3.4 || ^4.0",
         "symfony/config": "^3.4 || ^4.0",
+        "egulias/email-validator": "^3",
```

Comment:
This should be removed, it's just to test if this fixed https://app.travis-ci.com/github/silverstripe/silverstripe-framework/jobs/559512505